framework/src/st_things/things_stack/things_stack.c

```diff
@@ -376,6 +376,10 @@ int things_start_stack(void)
 			if (!things_network_turn_on_soft_ap()) {
 				return 0;
 			}
+			if (!things_wifi_scan_ap()) {
```

Comment:
I do not think you should scan for WiFi at this time.
You should scan the WiFi list when EasySetup starts.
Because the list of neighbor APs can change a lot after SoftAP is turned on.